pkg/server/server.go

```diff
@@ -586,6 +661,10 @@ func (s *Server) StreamedListObjects(req *openfgav1.StreamedListObjectsRequest,
 		s.checkResolver,
 		commands.WithLogger(s.logger),
 		commands.WithListObjectsDeadline(s.listObjectsDeadline),
+		commands.WithDispatchThrottler(s.listObjectsDispatchThrottler),
```

Comment:
nit (non-blocking), but please follow up - instead of passing all of these in individually, we should pass in a single thing:

```
dispatchConfig := DispatchConfig{
    Enabled: true,
    Throttler: s.listObjectsDispatchThrottler,
    Threshold: s.listObjectsDispatchThrottlingThreshold,
    MaxThreshold: s.listObjectsDispatchThrottlingMaxThreshold,
}
commands.WithDispatchConfig(dispatchCfg ...)
```

This applies to this, and other related opts where we're passing in 1 option per thing instead of passing in an option per category.

Reply:
Agreed 👍 handled these changes